packing_pin_util.rpt

```diff
@@ -0,0 +1,65 @@
+#Packing pin usage report
```

Comment:
Is this file accidentally added?  It looks like an output file that shouldn't go in the repo.